src/test/munit/flow-routing/flow-routing-server.xml

```diff
@@ -12,12 +12,6 @@
                  httpStatusVarName="httpStatus" parser="${parser.type}">
     <apikit:flow-mappings>
       <apikit:flow-mapping resource="/mapping" action="get" flow-ref="test-flow-mapping"/>
-
-      <!-- TODO: THIS ONES SHOULD BE REMOVED, SHOULD BE WAD -->
-      <apikit:flow-mapping resource="/apply-with-charset" action="put" content-type="application/json" flow-ref="put:\apply-with-charset:application\json:api-routing-config" />
```

Comment:
Do we have tests of flow-mapping ?

Reply:
Yes, we have the one that uses the flow-mapping that is above.